testdata/p4_16_samples_outputs/issue951-frontend.p4

```diff
@@ -0,0 +1,2 @@
+#include <core.p4>
+
```

Comment:
Why does this output not include the declaration?  Dead code elimination?

Reply:
Yes - in fact unused declaration elimination.
Previously it was also missing in --pp, before any dead code was executed.